packages/suite/src/middlewares/suite/analyticsMiddleware.ts

```diff
@@ -52,6 +53,7 @@ const analytics = (api: MiddlewareAPI<Dispatch, AppState>) => (next: Dispatch) =
                             .getState()
                             .devices.filter(d => d.remember && !d.useEmptyPassphrase).length,
                         theme: state.suite.settings.theme.variant,
+                        suiteVersion: process.env.version ? process.env.version : '',
```

Comment:
maybe more concise `process.env.version || ''`
btw. process.env.version is typed as `string | undefined`?

Reply:
The `process.env.version`  is `string | undefined` because of this interface 
![image](https://user-images.githubusercontent.com/44506010/104008023-21017a80-51a9-11eb-92a4-e01e39659d70.png)